### ISSUE TITLE
NullReferenceException. Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by AndreyZM** [bridgedotnet/Bridge#1813]
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at Bridge.Translator.SharpSixRewriter.VisitArgument(ArgumentSyntax node)
```

### Steps To Reproduce
http://deck.net/c1948b2d7f56af0c72f8f2f97a578e1e
```csharp
	public class Array
    {
  			public void Add(params object[] obj)
            {

            }
    }

    public class Program
    {
        public void Main()
        {
            Array callbacks = new Array();
           callbacks.Add(((System.Action<object>)(instance_callback)));
        }
        
        //remove static and its will compile
        static public void instance_callback(object a = null)
        {

        }
    }
```

**Comment by geoffreymcgill** [comment]
Thanks for the report and excellent sample demonstrating how to reproduce the issue. We are investigating and will fix right away.

How are you coming up with these scenarios?

**Comment by AndreyZM** [comment]
We are trying to port Flash/AS3 project to c# and compile it to js. This issues are introduced is Bridge 1.15 in 1.14 all this cases compiles well.

**Comment by geoffreymcgill** [comment]
This issue has been fixed and will be included in the Bridge 15.0.1 release which will be pushed to NuGet in a few minutes. 

Your original Deck now compiles properly. 

Thanks for reporting the issue.
